### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Jrmcuervo**
### Match Zeplin's design

In this PR I;

- Added any missing JSX elements.
- Matched the project to the Zeplin design.
- Make sure that there are no linter errors.
- Make sure that you used correct flow Gitflow.
- Make sure that you documented your work in a professional way.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+  margin: 1px;
+  font-family: "Roboto Slab", serif;
+}
+
+.book-details-text {
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #0290ff;
+  font-family: "Roboto Slab", serif;
+}
```
Comment (1GORDON) [inline src/App.css]: Great job @Jrmcuervo 🤝

- [ ] As stated in the project requirements, kindly do not repeat the same rule, instead create a single class for the repeating rule and reuse this class in your jsx code. Kindly impelement this for `font-family: "Roboto Slab", serif;`, `font-family: "Montserrat", sans-serif;`, and `flex`.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Jrmcuervo ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️
- Beautiful UI ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @SergioPeralta22 👋,

_This is an additional review as a result of the second opinion request._

For this project, your main objective is to style your website according to Zeplin design. But you are also required to adhere to HTML&CSS best practices, please check out the requirement below: 

image



The change requested by the previous reviewer was mentioned in [HTML&CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times) file (the point number 11): If you are using the same property for multiple elements, you can create a class attribute instead. For example, instead of using display: flex; 3 times:
```css
#intro1 {display: flex; font-size: 10px; font-weight: bold;}
header {display: flex; font-size: 20px; background-color: green;}
#banner {display: flex; font-size: 30px; background-image: url(images/static.jpg);}
```
you can create a class and add it to all element that need to be flex.
```
.flex {display: flex;}
```


Keep rocking 🚀!

**Comment by medaminedev66** [comment]
Hello @Jrmcuervo👋,

_This is an additional review as a result of the second opinion request._

For this project, your main objective is to style your website according to Zeplin design. But you are also required to adhere to HTML&CSS best practices, please check out the requirement below: 

image



The change requested by the previous reviewer was mentioned in [HTML&CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times) file (the point number 11): If you are using the same property for multiple elements, you can create a class attribute instead. For example, instead of using display: flex; 3 times:
```css
#intro1 {display: flex; font-size: 10px; font-weight: bold;}
header {display: flex; font-size: 20px; background-color: green;}
#banner {display: flex; font-size: 30px; background-image: url(images/static.jpg);}
```
you can create a class and add it to all element that need to be flex.
```
.flex {display: flex;}
```


Keep rocking 🚀!

**Review by alexander16108 — APPROVED** [review]
Hi @Jrmcuervo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
passing linters
good pr 
good readme file

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**